lib/teracy-dev-core/processors/variables.rb

```diff
@@ -41,6 +43,34 @@ def process(settings)
 
         settings
       end
+
+      private
+
+      def warning_for_unused_variables settings
+        return false if !TeracyDev::Util.exist? settings['variables']
+
+        extension_name_list = settings['teracy-dev']['extensions'].map { |x|
```

Comment:
this tight coupling should not be introduced, please follow my suggestion here instead https://github.com/teracyhq-incubator/teracy-dev-core/issues/51#issuecomment-433794390

Reply:
@hoatle I've updated, pls check again